source/includes/kubernetes/_k8_ingresses.md

```diff
@@ -1,13 +1,16 @@
 ### Ingresses V1Beta1
 
+Ingress v1beta1 has been deprecated in this API as of 26.7.0 following the removal of support by kubernetes. Please use the <a href="#kubernetes-ingresses-v1">ingress v1 api instead</a>.
+
 <!-------------------- LIST INGRESSES V1Beta1 -------------------->
+<!-- 
 
 #### List ingresses V1Beta1
 
 ```shell
 curl -X GET \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/api/v2/services/a_service/an_environment/ingresses"
```

Comment:
i am confused. I though we kept the /ingress for beta1.. changing would mean the client would need to change all their call.

Reply:
isn't it only the view builder that is changing?

Reply:
whoops it was confusing but you're right
reverted